metsim/constants.py

```diff
@@ -0,0 +1,56 @@
+"""
+Stores default constants, parameters, and options.  
+"""
```

Comment:
OK - opening an issue is fine - all I want here is separating true physical constants (stefan-boltmann, etc.) from the others